addons/account/models/account_move.py

```diff
@@ -2568,6 +2568,7 @@ def action_invoice_sent(self):
             message loaded by default
         """
         self.ensure_one()
+        self.check_access_rights('write')
```

Comment:
You also need to check the `ir.rule`s with `check_access_rule`[1], and since `access_account_move_salesman` only gives create access (and you probably shouldn't change that -- but I'm not certain), you have to check the `create` rights instead of `write` here

[1] the tricky part here is with `account_invoice_rule_see_personal`

Reply:
But the `group_sale_salesman` has neither `write` nor `create` access to `account_move`.
It's why I gave them the `write` access. 
https://github.com/odoo/odoo/pull/62188/files#diff-704354978d21157526c34c35900438b77010d9a5d4c92620aadc33f9e5d5ac56L9-R9
If i give them the `create` one, they will be able to create other invoices, even if they do not have access to Invoicing app (thanks to the `Create`button on the invoice to which they have the `read access`)

Ps: I will add the `check_access_rule` in the next commit but first I'm waiting to know if you agree with `write` access.